src/content.ts

```diff
@@ -207,6 +207,19 @@ async function setupFloatBar() {
     body {
         padding-bottom: ${ICON_SIZE}px;
     }
+
+    @media(prefers-color-scheme: dark) {
+        #engineSwitcherBar {
+            --bg: #000;
+            border-color: #111;
+        }
+        #engineSwitcherBar img[src$='wikipedia.svg']:not(:hover) {
+            filter: invert(100%);
+        }
+        #engineSwitcherBar a.active {
+            display: none;
```

Comment:
Why `display:none`??? Totally hide the active one?

Reply:
Maybe in light and dark we will now show the active button? Because when I am on Google - I know it... Or maybe add such option?

Reply:
Yes, I think you can add an option if you want such behavior. Because I personally still get used to the original behavior.